packages/devtools/lib/src/service_manager.dart

```diff
@@ -132,7 +132,7 @@ class ServiceConnectionManager {
 
     connectedApp = ConnectedApp();
 
-    service.onServiceEvent.listen((e) {
+    service.onEvent('_Service').listen((e) {
```

Comment:
nit: make a '_Service' a const since it is also used in the `streamIds` list

Reply:
done